### PR TITLE
[protocol] fold in com aspects

### LINKED ISSUE #3307: [com-to-io] Fold in com aspects of gitpod-protocol
All of our server API is OSS, and the current com aspects would be no different.

### PR DESCRIPTION
**Pull request by AlexTugarev**
fixes #3307

### CODE REVIEW

File: components/server/src/workspace/gitpod-server-impl.ts [hunk 1]
```diff
@@ -1605,4 +1608,93 @@ export class GitpodServerImpl<Client extends GitpodClient, Server extends Gitpod
         return this.termsProvider.getCurrent();
     }
 
+
+    //#region gitpod.io concerns
+    //
+    async adminGetAccountStatement(userId: string): Promise<AccountStatement> {
+        throw new ResponseError(ErrorCodes.EE_FEATURE, `Not implemented in OSS version`);
```
Comment (csweichel) [inline components/server/src/workspace/gitpod-server-impl.ts:1615]: We could introduce a separate error code for `SAAS` funtionallity.

```suggestion
        throw new ResponseError(ErrorCodes.SAAS_FEATURE, `Not implemented in this version`);
```

Comment (AlexTugarev) [reply components/server/src/workspace/gitpod-server-impl.ts:1615]: Let's go with SaaS

Comment (AlexTugarev) [reply components/server/src/workspace/gitpod-server-impl.ts]: done

**Review by svenefftinge — APPROVED** [review]
LGTM

**Pull request merged**